.github/workflows/test.yaml

```diff
@@ -67,3 +68,13 @@ jobs:
           --plugin_path /rbpf/target/release/examples/rbpf_plugin \
           --exclude_regex "${{ env.KNOWN_FAILURES }}"  \
           --plugin_options "--jit" || true
+
+      - name: Install grcov
+        run: |
+          cargo install grcov
```

Comment:
This step literally doubles the run time for the workflow. Would there be a way to download the binary instead of compiling it as part of the workflow? Looks like https://github.com/marketplace/actions/install-development-tools could be an option?